src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
```

Comment:
The instance of Random could be a class-level field instead of being created every time the 'getRandomBall' method is called. This would be more efficient as it avoids unnecessary object creation.

Reply:
fix comment